.github/workflows/build.yml

```diff
@@ -102,6 +104,16 @@ jobs:
         run: |
           echo "${{ secrets.DOCKER_BOT_TOKEN }}" | docker login -u dukeroboticsbot --password-stdin
           docker push ${IMAGE_NAME}
+      - name: Save onboard image
+        if: ${{ github.event.label.name == 'artifact' }}
```

Comment:
This was by design. My goal was for it to be a kind of switch, so that artifacts wont be created on every build, just certain ones. The other solution I was looking into was triggering the event on a comment, which is probably a better choice for this. I'll look into that